src/test/scala/com/redis/Bench.scala

```diff
@@ -31,8 +35,8 @@ object Bench {
 
   def load(opsPerClient: Int, fn: (Int, String) => Unit)(implicit clients: RedisClientPool): (Double, Double, Seq[_]) = {
     val start = System.nanoTime
-    val tasks = (1 to 100) map (i => actors.Futures.future { fn(opsPerClient, "k" + i.toString) })
-    val results = tasks map (future => future.apply())
+    val tasks = (1 to 100) map (i => Future { fn(opsPerClient, "k" + i.toString) })
+    val results = Await.result(Future.sequence(tasks), Duration.Inf)
```

Comment:
The point is the previous implementation was like this, no timeout. Does anybody know what the Banch class suppose to do?